docs/developers/guides/geth_js_console.md

```diff
@@ -0,0 +1,166 @@
+# Geth JavaScript Console Guide
+
+This is a comprehensive guide to setting up and using the Geth JS console with an Evmos node.
+
+## ✅ Requirements
+
+- Geth
+- Evmos (local) node
+
+### Installing Geth
+
+Install geth following the procedure corresponding to your OS in the [geth docs](https://geth.ethereum.org/docs/install-and-build/installing-geth).
+
+Check that the installation was successful by running the following command:
+
+```
+geth version
+```
+
+If everything went as expected, you should have an output similar to this:
+
+```
+Geth
+Version: 1.10.26-stable
+Git Commit: e5eb32acee19cc9fca6a03b10283b7484246b15a
+Architecture: amd64
+Go Version: go1.18.5
+Operating System: linux
+GOPATH=/home/tom/go
+GOROOT=/usr/local/go
+```
+
+## 1️⃣ Install dependencies
+
+Make sure you have installed all the dependencies mentioned in the **[Requirements](#-requirements)** section.
+
+## 2️⃣ Run Evmos local node
+
+- Clone the evmos repository (if you haven’t already)
+- Run the `local_node.sh` script to start a local node
+
+```
+git clone https://github.com/evmos/evmos.git
+cd evmos
+./local_node.sh
+```
+
+## 3️⃣ Attach geth JS console
+
+Wait a few seconds for the node to start up the JSON-RPC server.
+
+The local node has the HTTP-RPC server enabled and listening at port 8545 by default. This is what we will connect to.
+
+Attach geth console to your node with the following command:
+
+```
+geth attach http://127.0.0.1:8545
+```
+
+![https://i.imgur.com/rfN0T2i.png](https://i.imgur.com/rfN0T2i.png)
+
+## 4️⃣ Use JSON-RPC methods
+
+Now we can use all the implemented JSON-RPC methods. Find an exhaustive list of the supported JSON-RPC methods on [Evmos docs](https://docs.evmos.org/developers/json-rpc/endpoints.html).
+
+Below are some examples of how to use the console.
```

Comment:
Don't think that we need to use the numbers here, in the other developer guides we just use the headings.